### DISCUSSION TITLE
Bag. При редактирование топика не работают "Коды редактора"

### DISCUSSION THREAD

**Discussion by azarro** [madromas/esoTalk-v3#3] [Q&A] [answered]
При редактирование топика не работают "Коды редактора".  При добавление нового топика все работает исправно. Проверено на десктопе и андроиде. Язык русский.

**Comment by madromas** [comment]
Я вчера кое что правил, и заливал сюда. Поэтому у меня такой проблемы не наблюдается. Если у вас стоит esoTalk до вчерашнего дня, залейте по новому  "addons", "core" из архива. Ну и отпишитесь

**Reply by azarro** [reply]
Перезалил файлы - не помогло. Возможно вы меня не поняли. GIF прилагается. При редактировании топика, при нажатии на кнопки кодов, они не прописываются в редакторе. Пробовал на английской и русской версиях, в разных браузерах. Attach files тоже не срабатывает.

![1](https://github.com/madromas/esoTalk-v3/assets/2754781/23a916a6-0507-476f-b594-6e56aa1657f5)

**Reply by madromas** [reply]
какой у вас Browser? Отключите браузерные плагины, если таковы есть. На чистом firefox и edge, всё работает. На андройде тоже

**Reply by madromas** [reply]
И еще одно, почистете папку "Cache" в самой директории. Удалите там всё в папках css и js.

**Reply by madromas** [reply]
Да и если можно, какие плагины у вас включены в esoTalk?

Можете проверить тут https://hub.madway.net, при редактировании сбоего топика, работает или нет?

Я надеюсь у вас не влкючен "Debug" в конфигурации скрипта?

**Reply by azarro** [reply]
Проверял на Яндекс Браузере, Опера, Edge.  Папку Cache чистил.  Ctrl+F5

**Установленные плагины:**
1. Answers Allow posters to mark a reply as having answered their question.
2. Attachments Allows users to attach files to their posts.
3. AutoLink When you post an URL, AutoLinksLight automatically embeds videos from Youtube, Dailymotion, TwitchTV, RuTube, SoundCloud etc...
4. BBCode Formats BBCode within posts, allowing users to style their text.
5. ConfigEditor Edit the config.php file in the admin panel.
6. Conversation Status Allow those who can moderate to set a status to conversations.
7. Likes Allows members to like posts.
8. Shit A plugin enabling the label of shit posts
9. SpoilerTag Enable [spoiler] and [nsfw] tag to hide spoilers.
10. Views Counts the number of views of each conversation.

**Reply by azarro** [reply]
Да, у вас на сайте работает. Проверил. Могу дать данные от своего сайта посмотрите.

**Reply by azarro** [reply]
У себя на сайте включил Debug. Вот такая случилась беда. https://esotalk3.anclav24.keenetic.link

**Comment by madromas** [comment]
Debug в config.defaults.php, должен быть выключен!

$config["esoTalk.debug"] = **false**; // Debug mode will show advanced information in errors. Turn this off in production.

У вас на сайте регистрация отключена

**Reply by azarro** [reply]
Регистрацию включил.

**Reply by madromas** [reply]
Что то у вас там не так, Даже ответ не могу создать

**Comment by madromas** [comment]
Я сразу вижу что у вас в коде ошибки. Удалите всё и установите всё с начала.

**Reply by azarro** [reply]
ОК, 4 дня буду отсутствовать. Как приеду начну все с нуля.

**Reply by madromas** [reply, marked as answer]
Просто я ожень многое поменял на этой неделе. Какие именно файлы я менял уже не припомню, делел всё на автомате. Но было их много. Может какие забыл залить. К тому времени я надеюсь что всё поправлью :)
